packages/web-components/src/components/va-radio/va-radio.scss

```diff
@@ -0,0 +1,43 @@
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@use 'usa-form/src/styles/usa-form';
+@use 'usa-fieldset/src/styles/usa-fieldset';
+@use 'usa-legend/src/styles/usa-legend';
+@use 'usa-label/src/styles/usa-label';
+@use 'usa-hint/src/styles/usa-hint';
+@use 'usa-error-message/src/styles/usa-error-message';
+
+@import '../../mixins/uswds-error-border.css';
+
+:host {
+  display: block;
+  border: none;
+  padding: 0;
+}
+
+.usa-hint, .usa-error-message {
+  margin-bottom: 4px;
```

Comment:
Same question here.

Reply:
I didn't see an error message example on USWDS for radio and this matches what's in Sketch:

![Screen Shot 2022-11-22 at 1 05 51 PM](https://user-images.githubusercontent.com/872479/203399894-12ed7af0-8814-426b-8897-c1045fa3653f.png)

Reply:
This style was removed.